### ISSUE TITLE
Accessibility Audit

### ISSUE THREAD

**Issue by elioqoshi** [uracreative/works#95]

**Comment by axhakani** [comment]
## CalyxOS Accessibility Review

- In this phase, we reviewed the user flow and uncovered accessibility problems by different applications installed in the system.
- Then we did a detailed analysis on applications such as Datura Firewall, OpenKeychain, CalyxVPN and RiseupVPN. We identified common patterns that were confusing for the users including screen reader issues with instances of missing alt= tags.
- After that, we developed low-fidelity mockups, visualizing the best pathways to make the CalyxOS experience more accessible for the users.

## CalyxOS Review Summary

- Most of the buttons are called "unleveled button". For the user, their function is unknown and that leads to a problem. To help the user navigate freely, buttons need to be called by their name or their function.
- While using icons on a text, it would be best to include alt=tags of them. This way we can help the user understand the content of the text.

**Comment by axhakani** [comment]
Here you can follow the project on FigJam
https://www.figma.com/file/o2hXguWkAYIVCTPSJXWrrh/CalyxOS-Accessibility-Report?node-id=0%3A1
